docs/Universe/heroku.md

```diff
@@ -0,0 +1,88 @@
+title: Deploy Next.js starter to Heroku | Flotiq docs
+description: Deploy Next.js starter to Heroku
+
+# Deploy a NextJS to Heroku
+
+Flotiq Team prepared a few starters. They use frameworks like gatsby and NextJS.
+Such starters can be hosted on many popular, often free clouds.
+This post will guide you through the deployment process for one of the popular free clouds, Heroku.
+An example will be a starter written in the NextJS framework.
+
+## Before start
+To start, you need an account on Heroku and install Heroku CLI on your PC.
+
+The Heroku CLI is a command line tool to work with the Heroku cloud.
+The installation process is described on the page:
+
+[https://devcenter.heroku.com/articles/heroku-cli#install-the-heroku-cli](https://devcenter.heroku.com/articles/heroku-cli#install-the-heroku-cli)
+
+After installation is required login to Heroku via CLI:
+
+```bash
+heroku login
+```
+
+When we log in, the CLI tool is ready to go.
+## Select and fork starter
+
+Select one starter from Flotiq NetJs starter:
+[https://github.com/flotiq](https://github.com/flotiq)
+
+For example, next's steps show deployment starter nextjs blog 1.
+Fork this repository to your GitHub account.
+
+![Fork starter on GitHub](images/heroku/github-fork.png)
+
+After a fork, you need to clone the starter to your PC.
+
+```bash
+git clone git@github.com:ywur-account/flotiq-nextjs-blog-1.git
```

Comment:
Wrong word: ywur --> your

git clone git@github.com:ywur-account/flotiq-nextjs-blog-1.git
change to 
git clone git@github.com:your-account/flotiq-nextjs-blog-1.git